tests/toolchain/gcc_compilation.pm

```diff
@@ -0,0 +1,35 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
```

Comment:
you right I am just blindy copy/paste this part :) will change everything with my mind turned on :)